src/api/index.ts

```diff
@@ -147,3 +147,6 @@ function handleResponse<T>(response: AxiosResponse<T>) {
 export async function submitContactRequest(form: FormData) {
   return handleResponse(await api.post<string>(`/contact-us`, form));
 }
+export async function submitChangePasswordRequest(form: FormData) {
```

Comment:
You can type the parameter with the interface of the form (ex. {currentPassword, newPassword}). The only reason FormData was used on the contact.tsx page was because it included a file to be uploaded.